apps/web/playwright/fixtures/users.ts

```diff
@@ -229,12 +232,48 @@ export const createUsersFixture = (page: Page, workerInfo: WorkerInfo) => {
                 id: _user.id,
               },
             },
-            schedulingType: "COLLECTIVE",
-            title: "Team Event - 30min",
-            slug: "team-event-30min",
+            schedulingType: scenario.schedulingType ?? SchedulingType.COLLECTIVE,
+            title: teamEventTitle,
+            slug: teamEventSlug,
             length: 30,
           },
         });
+        if (scenario.teammates) {
+          // Create Teammate users
+          for await (const teammateObj of scenario.teammates) {
```

Comment:
```suggestion
          for (const teammateObj of scenario.teammates) {
```

I think await isn't needed as scenario.teammates doesn't have promises.